<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/res/background.png);
color:white;</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-20</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>331</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(47, 109, 156)</string>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="wendu">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="text">
       <string>32</string>
      </property>
     </widget>
     <widget class="QLabel" name="city">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>53</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>city</string>
      </property>
     </widget>
     <widget class="QLabel" name="type">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(44, 105, 150)</string>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="range">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(47, 109, 156)</string>
      </property>
      <property name="text">
       <string>range</string>
      </property>
     </widget>
     <widget class="QLabel" name="typeImage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>331</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(49, 109, 157);</string>
     </property>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>311</width>
        <height>131</height>
       </rect>
      </property>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(170, 85, 255);</string>
       </property>
       <widget class="QLabel" name="im1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>im1</string>
        </property>
       </widget>
       <widget class="QLabel" name="wind">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>wind</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(170, 85, 255);</string>
       </property>
       <widget class="QLabel" name="im2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>im2</string>
        </property>
       </widget>
       <widget class="QLabel" name="pm25">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>pm25</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_9" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(170, 85, 255);</string>
       </property>
       <widget class="QLabel" name="im3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>im3</string>
        </property>
       </widget>
       <widget class="QLabel" name="shidu">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>shidu</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_11" native="true">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(170, 85, 255);</string>
       </property>
       <widget class="QLabel" name="im4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>im4</string>
        </property>
       </widget>
       <widget class="QLabel" name="quality">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>quality</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="ganmao">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ganmao</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>400</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>632</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(40, 101, 146);
border: none;</string>
     </property>
     <property name="placeholderText">
      <string>城市</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background:rgb(194, 194, 194);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>431</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QLabel" name="day1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>03/16</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="day2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>day2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="day3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>day3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="day6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>day6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="day5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>day5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="day4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(1, 132, 156)</string>
      </property>
      <property name="text">
       <string>day4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="img1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img1</string>
      </property>
     </widget>
     <widget class="QLabel" name="img2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img2</string>
      </property>
     </widget>
     <widget class="QLabel" name="img3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img3</string>
      </property>
     </widget>
     <widget class="QLabel" name="img4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img4</string>
      </property>
     </widget>
     <widget class="QLabel" name="img5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img5</string>
      </property>
     </widget>
     <widget class="QLabel" name="img6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(42, 80, 107)</string>
      </property>
      <property name="text">
       <string>img6</string>
      </property>
     </widget>
     <widget class="QLabel" name="q1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>q1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="q2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>q2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="q3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>q3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="q4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>41</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="q5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>q1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="q6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>q1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="w4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(49, 111, 162)</string>
      </property>
      <property name="text">
       <string>wind</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="notice">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>340</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(53, 116, 163)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="download">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>400</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
background:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
